server.test.js

```diff
@@ -30,17 +35,99 @@ describe('Games', () => {
     // write a beforeEach hook that will populate your test DB with data
     // each time this hook runs, you should save a document to your db
     // by saving the document you'll be able to use it in each of your `it` blocks
+    const newGame = new Game({
+      title: 'fifa 2018',
+      genre: 'soccer',
+      releaseDate: 'JAN 2018'
+
+    });
```

Comment:
Because there is nothing that needs to be separated here and the blank space only separates closing brackets, I would move line 43 up to line 42 to keep the code together.